src/process.js

```diff
@@ -0,0 +1,174 @@
+import Data from './data.js';
+
+// get inputs from local storage
+
+export default class display {
+  static getToDoListFromStorage = () => {
+    let toDoLists;
+
+    if (JSON.parse(localStorage.getItem('LocalData')) === null) {
+      toDoLists = [];
+    } else {
+      toDoLists = JSON.parse(localStorage.getItem('LocalData'));
+    }
+    return toDoLists;
+  };
+
```

Comment:
This method is good but I think a better way to write this code will be to use a conditional statement. This will make your code DRY. This is an example in the link that you can use to understand this concept a lot better: [Conditional (ternary) Operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)